tests/publiccloud/check_registercloudguest.pm

```diff
@@ -19,7 +19,7 @@ use strict;
 use utils;
 use publiccloud::utils;
 use publiccloud::ssh_interactive 'select_host_console';
-use version_utils 'is_sle';
```

Comment:
You deleted the function definition.

Reply:
That's something that got my attention too, but the "use version_utils" is still there and this line was just inert.

Reply:
maybe it would be better remove `use version_utils;` and use a small explicit import of function? And check if aren't needed other exports from this module?

Reply:
Sorry: where do you see that `use` line still there? I cannot.

Reply:
@m-dati https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/22daf9cff82a2dc9ca048f8d74ab8d443ef15a5b/tests/publiccloud/check_registercloudguest.pm#L14

@mimi1vx fair, but I wasn't 100% sure if we use something else from the import, so it was "safer" to remove the explicit one.

Reply:
We actually use a bunch of different checks - `is_openstack, is_byos, is_sle` and more. It makes sense to just import the whole thing.

Reply:
agree , individual imports make sense when it is just 1,2 functions when amount  of imported functions grows over 5 IMHO small benefit of individual imports will be eaten by time needed to keep this list clean